apps/web/components/Organization/Wrapper/Themes/SyslumennTheme/SyslumennHeader.tsx

```diff
@@ -44,6 +47,14 @@ const SyslumennHeader: React.FC<React.PropsWithChildren<HeaderProps>> = ({
   const n = useNamespace(namespace)
   const { width } = useWindowSize()
 
+  const { activeLocale } = useI18n()
+  const logoAltText = n(
```

Comment:
Just in case there isn't a title for the organization perhaps it'd be better to fallback to an empty string instead of potentially concatenating undefined or '' with ' Frontpage'